### ISSUE TITLE
Tutorials

### ISSUE THREAD

**Issue by marioacalderon** [kittykatattack/hexi#77]
Hi, at the end of the readme, you promised more tutorials. My question is if you are going to create more have you abandoned the project?

My English is bad and that is why I apologize.

**Comment by malceore** [comment]
There hasn't been a commit on this repository since March 2018 so I wouldn't expect any new tutorials. But the **examples** section has **tons** of functioning games you can take apart and the subsections of the README have up to date documentation.

**Comment by marioacalderon** [comment]
Hello, thank you very much for your answer. Do you work with this framework?

**Comment by kittykatattack** [comment]
Yes, sorry, no new tutorials planned, unfortunately. I would love to write them, but time does not permit, sadly.
